src/server/session.rs

```diff
@@ -35,6 +36,8 @@ where
     pub data_cmd_rx: Option<Receiver<Command>>,
     pub data_abort_tx: Option<Sender<()>>,
     pub data_abort_rx: Option<Receiver<()>>,
+    pub internal_msg_tx: Option<Sender<InternalMsg>>,
+    pub connection: Option<ConnectionTuple>,
```

Comment:
perhaps good to specify what connection this is to quiet the questions e.g. proxy_connection or `data_connection`

Reply:
yes, in this case it's the control connection itself...  i'll see what i can do to make this more clear